tools/shared/symfun.h

```diff
@@ -234,6 +234,10 @@ inline void DTySet(DTYPE dtype, ISZ_T val) {
   stb.dt.stg_base[static_cast<int>(dtype)] = val;
 }
 
+inline void DTaSet(DTYPE dtype, ISZ_T val) {
+  stb.dt_align.stg_base[static_cast<int>(dtype)] = val;
```

Comment:
Instead of using a separate table, did you consider storing the alignment information in `dt` itself at an offset, e.g. `stb.dt.stg_base[static_cast<int>(dtype) + 6]`? Could you estimate the amount of effort that would require?